addons/qodot/src/util/qodot_texture_loader.gd

```diff
@@ -148,14 +155,21 @@ func create_material(
 	var texture : Texture = load_texture(texture_name)
 	if not texture:
 		return material
-
-	material.set_texture(SpatialMaterial.TEXTURE_ALBEDO, texture)
+	
+	if material is SpatialMaterial:
+		material.set_texture(SpatialMaterial.TEXTURE_ALBEDO, texture)
+	elif material is ShaderMaterial && default_material_albedo_uniform != "":
+		material.set_shader_param(default_material_albedo_uniform, texture)
 
```

Comment:
Just checking I'm reading this right, this section only handles setting the texture inside the material, right? Just checking that a ShaderMaterial without an albedo texture set still applies the ShaderMaterial as intended.

Reply:
Yes, that section only sets the texture for the material. When `default_material_albedo_uniform` is an empty string, the ShaderMaterial is still applied to the mesh, but it will use whatever default texture the material is set to. If the default uniform is not set, it will use the Godot default white texture. In the case of a shader that doesn't take a texture uniform, it should just work as normal.